plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/githubPullRequest.ts

```diff
@@ -298,7 +305,7 @@ export const createPublishGithubPullRequestAction = (
           changes: [
             {
               files,
-              commit: title,
+              commit: commitMessage || title,
```

Comment:
I think that you can use the null coalescence operator here.

```suggestion
              commit: commitMessage ?? title,
```